docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -206,4 +171,42 @@ PUT _application/search_application/my-app
   }
 }
 ----
-// TEST[skip:TBD]
+
+When the above `dictionary` parameter is specified, the <<search-application-search, search application search>> API will perform the following parameter validation:
+
+* It accepts only the `query_string` and `default_field` parameters
+* It verifies that `query_string` and `default_field` are both strings
+* It accepts `default_field` only if it takes the values `title` or `description`
+
+If the parameters are not valid, the the <<search-application-search, search application search>> API will return an error.
+[source,console]
+----
+POST _application/search_application/my-app/_search
+{
+  "params": {
+    "default_field": "author",
+    "query_string": "Jane"
+  }
+}
+----
+// TEST[catch:bad_request]
+
+In the above example, the value of the `default_field` parameter is not valid, therefore Elasticsearch will return an error:
+
+[source,console-result]
+----
+{
+  "error": {
+    "root_cause": [
+      {
+        "type": "validation_exception",
+        "reason": "Validation Failed: 1: $.default_field: does not have a value in the enumeration [title, description];"
+      }
+    ],
+    "type": "validation_exception",
+    "reason": "Validation Failed: 1: $.default_field: does not have a value in the enumeration [title, description];"
+  },
+  "status": 400
+}
+----
+// TEST[skip:example of a bad response]
```

Comment:
why does this need to be skipped?

Reply:
I time boxed this again this morning, this test will NOT pass. Something about malformed data. I think it's fine, here, because there are examples of other responses like this that we skip as it's an example, and we validate we get a bad request above.